gradient/cli/common.py

```diff
@@ -171,3 +219,31 @@ def options_file(f):
         )
     ]
     return functools.reduce(lambda x, opt: opt(x), reversed(options), f)
+
+class GradientObjectListOption(GradientOption):
+    def __init__(self, object_name, param_decls, **kwargs):
+        super(GradientObjectListOption, self).__init__(param_decls, **kwargs)
+        self.object_name = object_name
+        self.object_list_name = self.object_name + "s"
```

Comment:
This should not be a public field as it would break if the self.object_name is changed after creating the instance. Replace with a property:
@property
    def object_list_name(self):
        return self.object_name + "s"

https://docs.python.org/3/library/functions.html#property